guide/topics/discordjs/firstCommand.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Creating your first command
+authors:
+- NorteX (NorteX-dev)
+categories:
+JavaScript
+tags:
+- Discord.js
+- Bot Developement
+- Basics
+---
+
+Disclaimer: this tutorial assumes you've followed the [Getting Started](gettingStarted.md) tutorial.
```

Comment:
Based on checking all the Discord.js documentation links, I would include an additional disclaimer that this tutorial is based on the `stable` branch of Discord.js. It's not the biggest deal, but some people might want to know that knowledge before reading and if they need to double-check with the `master` branch before proceeding.